plugins/destination/file/client/read.go

```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/cloudquery/cloudquery/plugins/filetypes/csv"
+	"github.com/cloudquery/cloudquery/plugins/filetypes/json"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/google/uuid"
+)
+
+func (c *Client) ReverseTransformValues(table *schema.Table, values []any) (schema.CQTypes, error) {
+	switch c.pluginSpec.Format {
+	case FormatTypeCSV:
+		return c.csvReverseTransformer.ReverseTransformValues(table, values)
+	case FormatTypeJSON:
+		return c.jsonReverseTransformer.ReverseTransformValues(table, values)
+	default:
+		panic("unknown format " + c.pluginSpec.Format)
+	}
+}
+
+func (c *Client) Read(ctx context.Context, table *schema.Table, sourceName string, res chan<- []interface{}) error {
+	name := fmt.Sprintf("%s/%s.%s.%s", c.pluginSpec.Directory, table.Name, c.pluginSpec.Format, uuid.NewString())
```

Comment:
I don't see how this can read an existing file if it's generating a new UUID every time 🤔

Reply:
Added documentation under website. we use `NoRotate` in the tests

Reply:
I see, okay. We'll need a different solution when we start supporting cursor-based syncing, but since Read is only used in testing for now it's fine